app/screens/home/search/search.tsx

```diff
@@ -62,13 +71,9 @@ const SearchScreen = ({teamId}: Props) => {
         // - updated recent createdAt if exists??
 
         setLoading(true);
+        setFilter('all');
```

Comment:
should we make `all` a constant ? another option is set it to `undefined` which could have the same meaning

Reply:
I feel like it should remain as `all` because `all` is a selectable option in the UI. If there was instead a `clear filters` or `reset` button or something like that I think `undefined` would be appropriate.